### PR TITLE
Feature/signin card styling

### PR DESCRIPTION
**Pull request by mkazmer**
the Materialize formatting includes: `<div class="col s12 m6">`

this makes the card smaller and more square, but it doesn't resize very smoothly

If I remove it, it's a long rectangle, but it resizes cleaner

Looks like this EXCLUDING: `<div class="col s12 m6">`
Screen Shot 2019-05-29 at 6 49 23 PM

would love your thoughts!

### CODE REVIEW

**Comment by mkazmer** [comment]
@acousineau did some restyling of the sign in card....figure we can discuss tonight

File: client/src/components/auth/Signin.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      0 3px 1px -2px rgba(0, 0, 0, 0.12), 
+      0 1px 5px 0 rgba(0, 0, 0, 0.2);
+}
+
+.signin-card-title {
+  padding-left: 4%;
+  padding-top: 4%; 
+  font-size: 24px;
+  font-weight: 300;
+  font-weight: 400;
```
Comment (acousineau) [inline client/src/components/auth/Signin.scss]: Flub

File: client/src/components/auth/Signin.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.container {
```
Comment (acousineau) [inline client/src/components/auth/Signin.scss]: Delete - move to `.Signin` declaration

File: client/src/components/auth/SignIn.js [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+import './Signin.scss'
+
 const SignIn = props => {
   return (
-    <div className="">
-      <div className="">
-        <div className="">Please Sign In With Google</div>
-        <p className="">Users must log in with Google in order to use the application.</p>
-        <div className="">
+    <div className="signin-container">
```
Comment (acousineau) [inline client/src/components/auth/SignIn.js]: `.Signin` to match component name

File: client/src/components/auth/Signin.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  height: 100%;
+  width: 100%;
+}
+
+.signin-card {
```
Comment (acousineau) [inline client/src/components/auth/Signin.scss]: Nest this and the rest of the selectors inside of `.SignIn`

**Comment by mkazmer** [comment]
@acousineau so I'm guessing we will just want to close this since we are redoing the styling framework

I have just created a new branch for pulling out materialize

**Pull request closed without merging**